src/main/java/org/logstash/dissect/fields/AbstractField.java

```diff
@@ -0,0 +1,69 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.Delimiter;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public abstract class AbstractField implements Field {
+    private final int ordinal;
+    protected final String name;
+
+    protected Delimiter join;
+    protected Delimiter next;
```

Comment:
I am a little confused about what access modifier is needed to allow subclasses to access the superclass member.